internal/runbits/buildscript/file.go

```diff
@@ -40,8 +40,8 @@ func ScriptFromFile(path string) (*buildscript.BuildScript, error) {
 	return buildscript.Unmarshal(data)
 }
 
-func Initialize(path string, auth *authentication.Auth) error {
-	scriptPath := filepath.Join(path, constants.BuildScriptFileName)
+func Initialize(proj projecter, auth *authentication.Auth) error {
```

Comment:
It doesn't really make sense to me that this takes a project as an input. This story is effectively about introducing the project information to the buildscript, with the intend to eventually have this be the sole source of truth. For now we will also have this in the activestate.yaml, which is what the project package handles.

In order for this to not generate throwaway code or code that will need significant refactoring later, we would need to keep the buildscript and project package unaware of each other. And in order to expedite this we could use a mediator to handle mediating between the two (this is what I imagined `checkoutinfo` would be used for).

Passing in the project details as retrieved from the project package might be ok though. Just as long as buildscripts has zero awareness of the project package and vice versa. 

Basically think of how we want to use this package once its the source of truth, and write it that way. Then utilize the mediator to bridge the gap for now.

Reply:
Fair enough. Originally I added new owner and project string fields, but later realized the projector interface was available and provided that information, so I cleaned it up to do that. I've gone back to passing owner and project arguments again.